changelog/unreleased/bug-fixes/2896--remove-the-transformer-actor.md

```diff
@@ -0,0 +1,2 @@
+The transformer actor does not exist anymore. This resolves potential shutdown
+issues when using pipelines that contain aggregations.
```

Comment:
Hmm i wonder if we really need a changelog entry for this. Could someone more advanced comment on this? I think this issue might have been invisible to the users so we might as well just not bring it up.

Reply:
If the mentioned shutdown issues are potential in the sense of assumed, then I would indeed not bring it up in the changelog. I would also refrain from mentioning particular actors in these messages, as they are not elements that the reader understands. As a reader I would also like to see something akin to
```
Pipelines containing aggregations do not prevent VAST processes from terminating any more.
```
But again, only if the bug was actually confirmed.

Reply:
Instead of aggregations it is actually pipelines reducing the number of total events, but only when `vast.export.max-events` was set.

Reply:
So a better message would be:
```
Pipelines that reduce the number of events do not prevent `vast export` processes that have a `max-events` limit from terminating any more.
```

Reply:
I changed the changelogmessage to what you suggested in https://github.com/tenzir/vast/pull/2896/commits/b8a033309998307e58eea090684c888c146ec244.

Regarding the question whether this should be included at all: I don't know whether users in our target audiences have reported this bug, but I am also some kind of user and I discovered this bug quite easily.